beta-pipelines/multiome/atac.wdl

```diff
@@ -0,0 +1,186 @@
+version 1.0
+
+workflow ATAC {
+  meta {
+    description: "Processing for single-cell ATAC-seq data from the level of raw fastq reads. This is the first step of the multiome pipeline. ATAC-seq (Assay for Transposase-Accessible Chromatin using sequencing) is a technique used in molecular biology to assess genome-wide chromatin accessibility. This pipeline accepts fastq files where the cell barcode has been added to the fastq read names as the first field."
+    allowNestedInputs: true
+  }
+
+  input {
+    # Fastq inputs
+    File fastq_gzipped_input_read1
+    File fastq_gzipped_input_read2
+
+    # Output prefix/base name for all intermediate files and pipeline outputs
+    String output_base_name
+
+    # script for monitoring tasks 
+    File monitoring_script
+  }
+
+  parameter_meta {
+    fastq_gzipped_input_read1: "read 1 fastq file as input for the pipeline, the cellular barcodes must be the first part of the read name seperated by colon"
+    fastq_gzipped_input_read2: "read 2 fastq file as input for the pipeline, the cellular barcodes must be the first part of the read name separated by colon"
+    output_base_name: "base name to be used for the pipelines output and intermediate files"
+    monitoring_script : "script to monitor resource comsumption of tasks"
+  }
+
+  call TrimAdapters {
+    input:
+      fastq_input_read1 = fastq_gzipped_input_read1,
+      fastq_input_read2 = fastq_gzipped_input_read2,
+      output_base_name = output_base_name,
+      monitoring_script = monitoring_script
+    }
+
+  call BWAPairedEndAlignment {
+    input:
+      fastq_input_read1 = TrimAdapters.fastq_trimmed_adapter_output_read1,
+      fastq_input_read2 = TrimAdapters.fastq_trimmed_adapter_output_read2,
+      output_base_name = output_base_name,
+      monitoring_script = monitoring_script
+    }
+}
+
+  # trim read 1 and read 2 adapter sequeunce with cutadapt
+  task TrimAdapters {
+    input {
+      File fastq_input_read1
+      File fastq_input_read2
+      String output_base_name
+      String docker_image = "quay.io/broadinstitute/cutadapt:1.18"
+      File monitoring_script
+      Int disk_size = ceil(2 * ( size(fastq_input_read1, "GiB") + size(fastq_input_read2, "GiB") )) + 200
+      Int mem_size = 4
+      Int min_length 
```

Comment:
need to set a default or require it as a top level input

Reply:
Just to clarify, do we want to add the aligned bam file as an output at the workflow level?

Reply:
Ok sounds good!